### ISSUE TITLE
Caliban should be tested for accessibility

### ISSUE THREAD

**Issue by geneva-miller** [vanvalenlab/caliban#58]
Not sure what else this might cover but colorblindness is an obvious case. This may not be an immediate issue (for launching some Caliban jobs to annotators) but definitely should be explored.

**Comment by geneva-miller** [comment]
Since this issue was opened, additional display features and the use of React have improved the accessibility of the tool. I don't think there are any major accessibility issues with DCL at the moment, but this is something that should be kept in mind when developing and implementing new UI elements.

Some useful accessibility principles: https://www.w3.org/WAI/fundamentals/accessibility-principles/

A few of those principles that caught my eye include:
- not relying solely on color indicators to convey information
- use of alt-text/tooltips
- page navigation/interaction using a keyboard (although we expect DCL users to use a mouse or tablet for freehand drawing, keybinds and keyboard focus are still good to keep in mind)

**Comment by tddough98** [comment]
We use [mousetrap](https://craig.is/killing/mice) ([github](https://github.com/ccampbell/mousetrap)) for our keybinds. Keyboard focus is something that has some issues right now, where interacting with a slider or toggle will prevent mousetrap from receiving keyboard events and unintentionally disable keybinds until clicking elsewhere and resetting the keyboard focus. 

We can fix this with by adding the mousetrap class to the input components like 
```
  // Adds mousetrap class so keybinds work after using elements
  const inputRef = useRef();
  useEffect(() => {
    const input = inputRef.current;
    input.className = `${input.className} mousetrap`;
  }, []);
  
  ...
  
  return <ElementThatHogsKeyboardFocus inputRef={inputRef} />
  ```
  
Some older toggles have this fixed, but there are a number of new components components that break the keybinds that need to be fixed
